miasm2/ir/ir.py

```diff
@@ -256,15 +258,27 @@ def __init__(self, label, irs):
 
         assert isinstance(label, AsmLabel)
         self.label = label
-        self.irs = irs
-        self.except_automod = True
+        self._assignments = tuple(irs)
         self._dst = None
         self._dst_linenb = None
 
-    def _get_dst(self):
-        """Find the IRDst affectation and update dst, dst_linenb accordingly"""
-        if self._dst is not None:
-            return self._dst
+    @property
+    def irs(self):
+        return self._assignments
+
+    def is_dst_set(self):
+        return self._dst is not None
+
+    def updt_assignblock(self, index, assignblock):
```

Comment:
Same remark than before on `add`, `del`, ...
In addition, this API seems to be never used